src/main/target/FF_PIKOF4/target.c

```diff
@@ -28,18 +28,13 @@
 #include "drivers/timer_def.h"
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-#if defined(FF_PIKOF4OSD)
-    DEF_TIM(TIM5, CH4, PA3, TIM_USE_MOTOR, 0, 0 ), // S1_OUT - DMA1_ST7
-    DEF_TIM(TIM3, CH3, PB1, TIM_USE_MOTOR, 0, 0 ), // S2_OUT - DMA1_ST1
```

Comment:
This is swapping motor 2 and 4 outputs - this will lead to a lot of misconfigurations / user support tickets I suspect. This should be rearranged to keep the motor output order the same.

Reply:
I was working with FFPV on this and get confirmed a few times this is now correct. This also worred me how it cold work with former BF versions. Only have a non OSD board here for testing. @rex700 also tested this on a real board #7928. Requested a better expanation fro this from FFPV again.

Reply:
Thanks. I think we should hold off on mergeing this until we have an explanation - I don't want to think of the support carnage that would ensue if it turned out that the motor pins were swapped in a maintenance release...